examples/visualization/plot_image_gallery.py

```diff
@@ -23,18 +22,26 @@
     shuffle_files=True,
 )
 
+train_ds = train_ds.ragged_batch(16)
 
-def unpackage_tfds_inputs(inputs):
-    return inputs["image"]
+keras_cv.visualization.plot_image_gallery(
+    train_ds,
+    value_range=(0, 255),
+    scale=3,
+)
 
+"""
+If you want to use plain NumPy arrays, you can do that too:
+"""
+
+# Prepare some NumPy arrays from random noise
```

Comment:
This isn't random noise -- these are images from the tf.ds, right?

Reply:
Aha, yes. My bad, corrected it